ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/SqlLedgerSpec.scala

```diff
@@ -98,9 +108,13 @@ class SqlLedgerSpec
         )
 
       } yield {
-        ledger1.ledgerId should not be equal(LedgerId)
+        ledger1.ledgerId shouldBe a[LedgerId]
```

Comment:
Can this ever be false?

Reply:
You are right. on the other hand, the previous test was always true. might as well remove it.